internal/namesrv.go

```diff
@@ -125,3 +138,88 @@ func (s *namesrvs) SetCredentials(credentials primitive.Credentials) {
 func (s *namesrvs) AddrList() []string {
 	return s.srvs
 }
+
+// UpdateNameServerAddress will update srvs.
+// docs: https://rocketmq.apache.org/docs/best-practice-namesvr/
+func (s *namesrvs) UpdateNameServerAddress(nameServerDomain, instanceName string) {
+	if nameServerDomain == "" {
+		nameServerDomain = os.Getenv("NAMESRV_ADDR") // try to get from environment variable
```

Comment:
The value from os.Getenv("NAMESRV_ADDR") is the real name server address, not a domain server. 
it should not send a http request again.

Reply:
ok, i'll fix it